proposals/src/passive-stake-delegation-and-rewards.md

```diff
@@ -58,10 +58,27 @@ staker's StakeState accounts.  This is the percentage ceiling of the reward.
 * Account::lamports - The accumulated lamports from the commission.  These do not
 count as stakes.
 
-* `authorized_voter_id` - Only this identity is authorized to submit votes.
+* `authorized_vote_signer` - Only this identity is authorized to submit votes, and
+this field can only modified by this entity
 
+### VoteInstruction::Initialize
 
-### New StakeState
+* `account[0]` - RW - The VoteState
+  `VoteState::authorized_vote_signer` is initialized to `account[0]`
+   other VoteState members defaulted
+
+### VoteInstruction::AuthorizeVoteSigner(Pubkey)
+
+* `account[0]` - RW - The VoteState
+  `VoteState::authorized_vote_signer` is set to to `Pubkey`, instruction must by
+   signed by Pubkey
+
+
+### StakeState
+
+A StakeState takes one of two forms, StakeState::Delegate and StakeState::MiningPool.
```

Comment:
Why name the inner thing `Delegate`?

Reply:
because it captures the status of delegation and rewards collected via the node the stake is delegated to

Reply:
got it, thanks